config/model_configs/diagnostic_edmfx_trmm_box.yml

```diff
@@ -27,3 +27,4 @@ dt: 10secs
 t_end: 6hours
 dt_save_to_disk: 10mins
 FLOAT_TYPE: "Float64"
+toml: [toml/diagnostic_edmfx_trmm_box.toml]
```

Comment:
```suggestion
toml: [toml/diagnostic_edmfx_trmm_box.toml]
override_τ_precip: false
```

Reply:
Just to make sure τ_precip gets set properly!

Reply:
Right, thanks!